.github/workflows/wheel_win_x64.yml

```diff
@@ -1,8 +1,6 @@
 name: Wheel build - Windows CPU x86_64
 on:
   workflow_dispatch: # allows triggering the workflow run manually
-  release:
```

Comment:
This is an intentional change. We need to trigger a wheel build before the github release is pushed, not after.